contrib/pmsnare/pmsnare.c

```diff
@@ -81,11 +84,10 @@ CODESTARTisCompatibleWithFeature
 		iRet = RS_RET_OK;
 ENDisCompatibleWithFeature
 
-
 BEGINparse
 	uchar *p2parse;
 	int lenMsg;
-	int snaremessage;
+	int len_snare_tag;
 	int tablength;
 
 CODESTARTparse
```

Comment:
hmm, thinking out loud, can we just define a variable to contain the string "\t" and then have the escaping do whatever conversion it is doing to the message to this variable as well? if we can do that, then it should be almost as easy as changing TabRepresentation from being an #define to being this variable (there is the checking that a variable works everywhere that currently has a macro defining a constant string, and changing things to dereference as needed)